integreat_compass/cms/templates/offers/my_offers_list.html

```diff
@@ -0,0 +1,67 @@
+{% extends "_base.html" %}
+{% load i18n %}
+{% load static %}
+{% load widget_tweaks %}
+{% load render_bundle from webpack_loader %}
+{% block title %}
+    {% translate "My offers" %}
+{% endblock title %}
+{% block content %}
+    <div>
+        <h1 class="mb-8">{% translate "My offers" %}</h1>
+        {% for offer in object_list %}
+            <form method="post"
+                  action="{% url 'cms:protected:delete_offer' offer.pk %}">
+                {% csrf_token %}
+                <input type="hidden" name="pk" value="{{ offer.pk }}">
+                <div class="md:flex flex-row border-2 justify-between border-gray-200 rounded-lg shadow-md shadow-inner mb-8 gap-6 overflow-hidden">
+                    <div class="basis-1/8 shrink-0">
+                        <img class="object-cover h-[300px]"
+                             src="{% if offer.latest_version.title_image and offer.latest_version.title_image.url %}{{ offer.latest_version.title_image.url }}{% else %}{% static fallback_title_image %}{% endif %}"
```

Comment:
```suggestion
                             src="{% if offer.latest_version.title_image %}{{ offer.latest_version.title_image.url }}{% else %}{% static fallback_title_image %}{% endif %}"
```
If `offer.latest_version.title_image` evaluates to `True`, it is guaranteed that `offer.latest_version.title_image.url` does as well.

Also, I think we will probably need this expression multiple times, so why not add a property for this as suggested in https://github.com/digitalfabrik/integreat-compass/pull/71#discussion_r1252862820?
(You can also use the template tags in views by importing them `from django.templatetags.static import static`, see [SO](https://stackoverflow.com/questions/11721818/django-get-the-static-files-url-in-view))

Reply:
Yep, good idea.